enzyme/test/Enzyme/ForwardMode/blas/cblas_ddot.ll

```diff
@@ -1,5 +1,5 @@
-; RUN: if [ %llvmver -lt 16 ]; then %opt < %s %loadEnzyme -enzyme -S | FileCheck %s; fi
-; RUN: %opt < %s %newLoadEnzyme -passes="enzyme" -enzyme-preopt=false -S | FileCheck %s
+; RUN: if [ %llvmver -lt 16 ]; then %opt < %s %loadEnzyme -enzyme -enzyme-runtime-activity=1 -S | FileCheck %s; fi
+; RUN: %opt < %s %newLoadEnzyme -passes="enzyme" -enzyme-preopt=false -enzyme-runtime-activity=1 -S | FileCheck %s
```

Comment:
@ZuseZ4 we should error if forward mode runtime activity is requetsed (rather than silently doing the wrong thing, like here).